source/reading.md

```diff
@@ -964,637 +909,637 @@ Databases are beneficial in a large-scale setting:
 ```
 
 At the middle and end of a data analysis, we often want to write a data frame
-that has changed (either through filtering, selecting, mutating or summarizing)
+that has changed (through selecting columns, filtering rows, etc.)
 to a file to share it with others or use it for another step in the analysis.
 The most straightforward way to do this is to use the `to_csv` function
 from the `pandas` package.  The default
-arguments for this file are to use a comma (`,`) as the delimiter and include
-column names. Below we demonstrate creating a new version of the Canadian
-languages data set without the official languages category according to the
+arguments are to use a comma (`,`) as the separator, and to include column names. 
+Below we demonstrate creating a new version of the Canadian
+languages data set without the "Official languages" category according to the
 Canadian 2016 Census, and then writing this to a `.csv` file:
 
 ```{code-cell} ipython3
-no_official_lang_data = canlang_data[canlang_data['category'] != 'Official languages']
+no_official_lang_data = canlang_data[canlang_data["category"] != "Official languages"]
 no_official_lang_data.to_csv("data/no_official_languages.csv")
```

Comment:
I think we should recommend `index=False` because otherwise the index is written to the CSV, which is rarely desired and an odd default imo.

Reply:
ah agree -- I had this in mind when I was working but then forgot to actually do it ;) will fix

Reply:
fixed